src/lightning_app/cli/lightning_cli.py

```diff
@@ -206,7 +208,24 @@ def stop():
     pass
 
 
-@_main.group(hidden=True)
+@stop.command("app")
+@click.argument("id", type=str)
```

Comment:
I am not sure if this is the right UX though. ID is not a memorizable string. So users need to get to the UI to copy the id to execute this command, which essentially defeat the purpose no?

Why wouldn't we use app name here?

Reply:
The main reason to not use app name here is because the app name is not guaranteed to be unique. We are also adding the ability for the CLI to list apps, which could return their IDs to be used to delete a given app without using the UI.

Reply:
The name is guarenteed to be unique under each project. We could raise a warning if user has multiple apps with same name but otherwise delete the one with the name provided?

Reply:
Hmm, the BE List API requires a project ID to be passed always which makes the FE logic a bit complex. Perhaps this could be a feature for future